### PR TITLE
Use the new user-access API

### PR DESCRIPTION
**Pull request by pjcdawkins**
- [x] Depends on https://github.com/platformsh/platformsh-client-php/pull/88

### CODE REVIEW

File: src/Command/User/UserAddCommand.php [hunk 1]
```diff
@@ -118,63 +118,94 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+                $selection = $this->loadProjectUser($project, $userId);
+                if (!$selection) {
+                    throw new InvalidArgumentException('User not found: ' . $userId);
                 }
-                $email = $questionHelper->choose($choices, 'Enter a number to choose a user to update:');
             } else {
                 $question = new Question("Enter the user's email address: ");
                 $question->setValidator(function ($answer) {
-                    return $this->validateEmail($answer);
+                    if (empty($value)) {
```
Comment (eflies) [inline src/Command/User/UserAddCommand.php:142]: `empty($answer)`

File: src/Command/User/UserAddCommand.php [hunk 2]
```diff
@@ -118,63 +118,94 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         // Process the [email] argument.
-        $email = $input->getArgument('email');
-        if (!$email) {
+        $email = null;
+        if ($emailOrId = $input->getArgument('email')) {
```
Comment (eflies) [inline src/Command/User/UserAddCommand.php:122]: Shouldn't we set `$email` somewhere in this block? Right now we are sending empty value on invitations

Comment (pjcdawkins) [reply src/Command/User/UserAddCommand.php:125]: Yes, thanks

**Comment by bojanz** [comment]
I've tested the new commands manually, and confirmed their requests using `-vvv`. So far so good.

**Pull request merged**